lib/src/chart/radar_chart/radar_chart_helper.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:equatable/equatable.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:fl_chart/src/utils/list_wrapper.dart';
+
+/// Contains all the function to help the RadarChart work
+class RadarChartHelper {
+  /// Contains List of cached results, based on [List<RadarDataSet>]
+  ///
+  /// We use it to prevent redundant calculations
+  static final Map<ListWrapper<RadarDataSet>, RadarChartMinMaxAxisValues>
+      _cachedResults = {};
```

Comment:
This cache is never cleaned.
This could lead to a memory leak

Reply:
That's a great point. We have similar functionality like this in our LineChart and BarChart. 
The point is that as we recreate our chart data classes, we need to recalculate these values again and again. That's why I implemented this logic to handle a simple cache. And here, @DooRooSA followed what I did before.